local_settings.py.tmpl

```diff
@@ -315,3 +315,6 @@ REQUEST_IGNORE_PATHS = (
     r'^admin/',
```

Comment:
Middleware setup seems missing.

Reply:
Fix with this commit https://github.com/gem/oq-platform2/pull/112/commits/b0caf6ed7355b191c0d216e5fcedf0c96527cdbc